renderdoc/driver/metal/metal_device.cpp

```diff
@@ -38,3 +51,121 @@ MTL::Device *WrappedMTLDevice::MTLCreateSystemDefaultDevice(MTL::Device *realMTL
 
   return UnwrapObjC<MTL::Device *>(wrappedMTLDevice);
 }
+
+MTL::Library *WrappedMTLDevice::newDefaultLibrary()
+{
+  MTL::Library *realMTLLibrary;
+
+  SERIALISE_TIME_CALL(realMTLLibrary = Unwrap<MTL::Device *>(this)->newDefaultLibrary());
+  MetalResourceManager::UnwrapHelper<MTL::Library *>::Outer *wrappedMTLLibrary;
```

Comment:
This kind of template instantiation makes it harder than necessary to see what the type is - it should instead just be `WrappedMTLLibrary`.

Reply:
1f9a217577fe36c8bc218da0bbb2f3e721f22011